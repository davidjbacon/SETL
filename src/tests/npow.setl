PROGRAM t;
  a := 3; b := <<2,11,113,5709>>; c := <<7>>;
  $ try changing BOP to + and see the output change:
  PRINT (a NPOW b BOP c);
  PRINT ((a NPOW b) BOP c);
  PRINT (a NPOW (b BOP c));
  $ and we assume LESS is like WITH
  OP BOP(j,k);
    RETURN (j+k);
  END;
END;
