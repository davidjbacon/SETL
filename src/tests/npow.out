{7 {2 11 5709} {2 11 113} {2 5709 113} {11 5709 113}}
{7 {2 11 5709} {2 11 113} {2 5709 113} {11 5709 113}}
{{2 7 11} {2 7 5709} {2 7 113} {2 11 5709} {2 11 113} {2 5709 113} {7 11 5709} {7 11 113} {7 5709 113} {11 5709 113}}
